.zenodo.json

```diff
@@ -249,5 +249,5 @@
   "license": {
     "id": "AGPL-3.0-or-later"
   },
-  "publication_date": "2023-07-13"
+  "publication_date": "2024-03-27"
```

Comment:
didn't you use 2024-03-28 in the other files? https://github.com/remindmodel/remind/blob/develop/CHANGELOG.md#330---2024-03-28